### PR TITLE
[RDY] Add the Summary

### PR DESCRIPTION
**Pull request by Hjdskes**
Branched from #77.

It's a mixup between the project plan, preface and introduction.

### CODE REVIEW

**Comment by Hjdskes** [comment]
Todo: add all contributions made by this project.

File: final/summary.tex [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+evaluates them and prints the result(s). REPLs facilitate exploratory
+programming and debugging and are therefore a nice addition to Spoofax.
+
+This report comprises the end of the TU Delft Computer Science Bachelor Project;
+the final, compulsory project at the Delft University of Technology to attain
+the Bachelor of Science in Computer Science.
+
+The purpose of this report is to describe the project as it was carried out over
+the course of ten weeks. Not only does it describe the process, it also aims to
+inform the reader about the problem and the design and implementation of the
```
Comment (Balletie) [inline final/summary.tex]: maybe:

> the problem, the design and implementation

Comment (Hjdskes) [inline final/summary.tex]: Design and implementation go together, don't they?

Comment (Balletie) [inline final/summary.tex]: > the problem as well as the design and the implementation

**Pull request merged**